src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -58,11 +64,28 @@ const props = defineProps({
     type: String,
     default: 'Clear filters',
   },
+  areFiltersHidden: {
+    type: Boolean,
+    default: () => undefined,
+  },
+  hiddenFiltersNames: {
+    type: Array,
+    default: () => [],
+  },
 });
 
 const emit = defineEmits(['update:modelValue', 'change']);
+const areFiltersHiddenThreshold = useMediaQuery('(max-width: 768px)');
```

Comment:
I think the toggling of visibility should be handled by the parent component when used.

Reply:
By doing that I think that in this case we can also reuse the [`useMediaQuery` from `ec-table`](https://github.com/Ebury/chameleon/blob/master/src/components/ec-table/ec-table.vue#L83) as well since it's the same threshold, wdyt?

Reply:
as soon as we introduce isHidden prop to every filter, then we let user of this component let to decide when and what filters to hide.